test/controllers/EditSetControllerSpec.scala

```diff
@@ -945,6 +946,18 @@ class EditSetControllerSpec extends BaseSpec {
               ExpectedSelectOption("1", "The National Archives, Kew", selected = true),
               ExpectedSelectOption("2", "British Museum, Department of Libraries and Archives"),
               ExpectedSelectOption("3", "British Library, National Sound Archive")
+            ),
+            relatedMaterial = Seq(
+              ExpectedRelatedMaterial(
+                linkHref = "#;",
```

Comment:
I assume that's what "At present the links will not de-referenceable" means - these links do not link anywhere at the minute. I assume in the future they can link to any kind of record or edit set.

Reply:
Exactly right @felixgb